packages/runtime/src/next/components/tests/page-date-prop-controller.test.tsx

```diff
@@ -0,0 +1,123 @@
+/** @jest-environment jsdom */
+
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+
+import { ElementData } from '../../../state/react-page'
+import { randomUUID } from 'crypto'
+import {
+  Date as DatePropController,
+  DateDescriptor,
+  Types,
+  createDatePropControllerDataFromString,
+} from '@makeswift/prop-controllers'
+import { Page } from '../page'
+import { act } from 'react-dom/test-utils'
+import { ReactRuntimeProvider } from '../../context/react-runtime'
+import { ReactRuntime } from '../../../react'
+import { forwardRef } from 'react'
+import {
+  createMakeswiftPageSnapshot,
+  createRootComponent,
+} from '../../../utils/tests/element-data-test-test'
+
+describe('Page', () => {
+  test('can render DatePropController v0 data', async () => {
+    // Arrange
+    const dateDefinitionV0: DateDescriptor = {
+      type: Types.Date,
+      options: {},
+    }
+    const dateData = new Date(Date.now()).toISOString()
```

Comment:
`new Date(Date.now())`? Is this Copilot? Why not just `new Date()`?

Reply:
I didn't use copilot when writing this PR. This was a human mistake. I tried `Date.now().toISOString()` and it didn't work 🤦 

Updated to be:
```ts
new Date().toISOString()
```